CONFIGURATION.md

```diff
@@ -11,6 +11,19 @@ Here are some Stargate-relevant property groups that are necessary for correct s
 * `quarkus.grpc.clients.bridge` - property group for defining the Bridge gRPC client (see [gRPC Client configuration](https://quarkus.io/guides/grpc-service-consumption#client-configuration) for all options)
 * `quarkus.cache.caffeine.keyspace-cache` - property group  for defining the keyspace cache used by [SchemaManager](../sgv2-quarkus-common/src/main/java/io/stargate/sgv2/api/common/schema/SchemaManager.java) (see [Caffeine cache configuration](https://quarkus.io/guides/cache#caffeine-configuration-properties) for all options)
 
+
+## Document limits configuration
+*Configuration for document limits, defined by [DocumentLimitsConfig.java](src/main/java/io/stargate/sgv2/jsonapi/config/DocumentLimitsConfig.java).*
+
+| Property                                                 | Type  | Default     | Description                                                                                                 |
+|----------------------------------------------------------|-------|-------------|-------------------------------------------------------------------------------------------------------------|
+| `stargate.jsonapi.document.limits.max-size`              | `int` | `1_000_000` | The maximum size of a single document in characters. Defaults to 1 million characters or approximately 1MB. |
+| `stargate.jsonapi.document.limits.max-depth`             | `int` | `8`         | The maximum document depth (nesting).                                                                       |
+| `stargate.jsonapi.document.limits.max-name-length`       | `int` | `48`        | The maximum length of property names in a document for an individual segment.                               |
```

Comment:
@tatu-at-datastax this is a bit confusing for me, should we go for one of:

1. `stargate.jsonapi.document.limits.max-property-length`
2. `stargate.jsonapi.document.limits.max-promerty-name-length`

Reply:
Yes I'd like "name" to be included (in case we also have property value lengths) so (2)

`stargate.jsonapi.document.limits.max-property-name-length`

sounds good to me.

Reply:
Done.